server/events/project_command_context_builder.go

```diff
@@ -121,7 +121,12 @@ func (cb *DefaultProjectCommandContextBuilder) BuildProjectContext(
 		parallelPlan,
 		verbose,
 		ctx.Scope,
-	))
+	)
+
+	// Map the CommandContext PullReqStatus to ProjectCommandContext PullReqStatus.
+	projectCmdContext.PullReqStatus = ctx.PullRequestStatus
+
+	projectCmds = append(projectCmds, projectCmdContext)
```

Comment:
Why are we using a slice here if we're only appending once? @nishkrishnan Maybe I missed something!

Reply:
🤷🏽